apps/server/src/modules/medium-metadata/service/medium-metadata.service.ts

```diff
@@ -0,0 +1,45 @@
+import { MediaSourceDataFormat, MediaSourceRepo } from '@modules/media-source';
+import { Injectable } from '@nestjs/common';
+import { MediumMetadataDto } from '../dto';
+import {
+	MediaSourceDataFormatNotFoundLoggableException,
+	MediaSourceNotFoundLoggableException,
+	MediumMetadataStrategyNotImplementedLoggableException,
+} from '../loggable';
+import { BiloStrategy, MediumMetadataStrategy, VidisStrategy } from '../strategy';
+
+@Injectable()
+export class MediumMetadataService {
+	private mediumMetadataStrategyMap: Map<MediaSourceDataFormat, MediumMetadataStrategy>;
+
+	constructor(
+		private readonly mediaSourceRepo: MediaSourceRepo,
```

Comment:
You shouldn't use MediaSourceRepo outside of the MediaSourceModule. Please use the MediaSourceService and remove the export of the repo from the MediaSourceModule.